ActivityTracker/background.js

```diff
@@ -161,16 +175,17 @@ class StatisticsHandler {
         }
         hostTimeData.activate();
         hostTimeData.addTab(tabId);
+        this.currentTab = tabId;
         this.lastHostname = hostname;
     }
 
     deactivate(tabId) {
         console.log('Deactivate statistics timer for tabId: ' + tabId);
         for (let host in this.hostnameToTimeData) {
-        	const value = this.hostnameToTimeData[host];
-            console.log('Opened tabs: ' + this.hostnameToTimeData[host].tabIds);
+            const value = this.hostnameToTimeData[host];
             if (value.tabIds.has(tabId)) {
                 console.log('User closed tab with hostname: ' + host);
+                console.log('Opened tabs: ' + Array.from(this.hostnameToTimeData[host].tabIds).toString());
```

Comment:
you don't need to call  `.toString()` explicitly. It will be called when you concatenate an array with a string.